### PR TITLE
Fix: removed space before extends keyword on class declaration

### PR DESCRIPTION
**Pull request by Shinyaigeek**
## Motivation

Fix: #1764 

I fixed the error that with minify option and es2016 ~ 2020 target option `swc` outputs invalid javascript code due to removed space before `extends` keyword on class declaration.

if something wrong (incorrect approach or not enough test case...), please feel free to close this PR or point it out 🙏

### CODE REVIEW

**Review by kdy1 — APPROVED** [review]
Thanks!

**Comment by Shinyaigeek** [comment]
Oops, some tests is failed :bow:
I am investigating why.

**Comment by kdy1** [comment]
UPDATE=1 cargo test --all

will fix it I think.
It's bit slow, but it updates all specs of all tests.

File: ecmascript/codegen/src/lib.rs [hunk 1]
```diff
@@ -835,6 +835,7 @@ impl<'a> Emitter<'a> {
     #[emitter]
     fn emit_class_trailing(&mut self, node: &Class) -> Result {
         if node.super_class.is_some() {
+            space!();
```
Comment (Shinyaigeek) [inline ecmascript/codegen/src/lib.rs:838]: I got my approach was wrong. This updated code will insert space between `class ident` and `extends keyword`, but when minify option is false, space already exists after `class ident`, so when minify option is false, two spaces are inserted between  `class ident` and `extends keyword`.  I will fix this.

**Comment by Shinyaigeek** [comment]
> UPDATE=1 cargo test --all

@kdy1 OK, I will try it!! 👌  thank you!

**Comment by Shinyaigeek** [comment]
@kdy1 I fix the approach(eventually I did not touch tests), please check it when you are free 🙇

**Pull request merged**